### ISSUE TITLE
Figure description error

### ISSUE THREAD

**Issue by michelonfrancoisSUMMIT** [arXiv/html_feedback#4043]
### Description

The tag \Description of a figure is not taken into account in the HTML rendering.

### (Required) What is this issue most closely related to? Select one.

Choose One

### Internal issue ID

8ea2c137-5d45-4267-8200-70a83d580c65

### Paper URL

https://arxiv.org/html/2410.17290v1

### Browser

Firefox/139.0

### Device Type

Desktop

**Comment by dginev** [comment]
Bookkeeping comment: This is pending the upgrades from https://github.com/brucemiller/LaTeXML/pull/2666

**Comment by dginev** [comment]
Thank you for your report - the specific issue is now solved.

We have regenerated the article and you should be able to see an improvement.
